ci-operator/config/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-master.yaml

```diff
@@ -43,6 +51,9 @@ tests:
   commands: make test
   container:
     from: src
+- as: e2e-hypershift
+  steps:
```

Comment:
Could we add `optional: true` until we know this doesn't introduce any issues for cluster-ingress-operator CI?

Reply:
We are actually intentionally setting as not optional at this point we GAing to avoid operators breaking us at it has happened in the past.
We are doing the same in multiple operators and we are also ci stream promotion blocker now. I think it's a fair organic step and we can always relax it back if it cause unncessary issues. Makes sense?

Reply:
This is the newest example https://github.com/openshift/cluster-ingress-operator/pull/930.
We need to get this merged asap right after fixing this new issue so teams are able to operate with healthy safety checks that ensure GA level stability.